packages/design-picker/src/components/generated-design-picker.tsx

```diff
@@ -1,11 +1,11 @@
 /* eslint-disable wpcalypso/jsx-classname-namespace */
-
 import { Button } from '@automattic/components';
 import { MShotsImage } from '@automattic/onboarding';
 import { useViewportMatch } from '@wordpress/compose';
 import { useI18n } from '@wordpress/react-i18n';
 import classnames from 'classnames';
-import { useEffect, useRef } from 'react';
+import { useEffect, useRef, useState } from 'react';
+import PreviewToolbar from '../../../../client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/preview-toolbar';
```

Comment:
I don't think we can import PreviewToolbar as it's in the client codebase instead of any packages 😞

Reply:
Yes, since `design-picker` it's a package, consider it as isolated as the rest of calypso's code.